ios/Plugin/TwilioVideoViewController.swift

```diff
@@ -58,6 +59,8 @@ class TwilioVideoViewController: UIViewController {
         self.disconnectButton.isHidden = true
         self.micButton.isHidden = true
 
+        self.statusLabel.text = "Connecting"
```

Comment:
not translated? if in android it is translated, it should be also here, if not, follow up task so we don't forget